### PR TITLE
Docs: create cytoscape example

### PR DESCRIPTION
**Pull request by celine168**
Documents the use of ipycytoscape, based on @moorepants's PR for documenting ipythreejs.

### CODE REVIEW

**Comment by moorepants** [comment]
Cool. Add this to the toctree in the index.rst file.

**Comment by moorepants** [comment]
This works for me locally:
image

File: docs/ipycytoscape_example.rst [hunk 1]
```diff
@@ -0,0 +1,137 @@
+====================
+IPyCytoscape Example
+====================
+
+Thebe can display output from ipycytoscape_, which could visualize graphs using Cytoscape.js_.
```
Comment (moorepants) [inline docs/ipycytoscape_example.rst:5]: Mention the license of the code you copied from ipycytoscape (if you did) like we did in the other examples.

File: docs/ipycytoscape_example.rst [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+Setup
+=====
+
+Be sure to load require.js before any of your thebe activation code so that the
+Cytoscape visualizations can function:
+
+.. code-block:: html
+
+   <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js"></script>
```
Comment (moorepants) [inline docs/ipycytoscape_example.rst:18]: Did you verify that this is required? (no pun intended)

**Comment by moorepants** [comment]
@celine168 If you make the two updates, this can be merged.

File: docs/ipycytoscape_example.rst (hunk 2 above)
Comment (celine168) [reply docs/ipycytoscape_example.rst:18]: Yes, this is required, or the widget fails to load.
image

This is from the same code except removing require.js.

File: docs/ipycytoscape_example.rst (hunk 1 above)
Comment (celine168) [reply docs/ipycytoscape_example.rst]: Added the mention to the BSD-3 Clause license.

File: docs/ipycytoscape_example.rst (hunk 2 above)
Comment (moorepants) [reply docs/ipycytoscape_example.rst]: Thanks. It would be nice to know which ones require require.js. Seems like all of the ones we've tested so far.

**Comment by moorepants** [comment]
LGTM!

**Pull request merged**